README.md

```diff
@@ -31,7 +31,7 @@ You can check the detailed [change log](https://github.com/brightcove/hot-shots/
 All initialization parameters are optional.
 
 Parameters (specified as one object passed into hot-shots):
-* `host`:        The host to send stats to, if not set, the constructor tries to retrieve it from the `DD_AGENT_HOST` environment variable, `default: localhost`
+* `host`:        The host to send stats to, if not set, the constructor tries to retrieve it from the `DD_AGENT_HOST` environment variable, `default: 127.0.0.1`
```

Comment:
Can you change this to indicate that now default: '127.0.0.1' or '::1'

Reply:
Sure, applied the change. Should we not set it to `undefined` with a note on dram socket's behaviour?

Reply:
That's fine as well, to note it that way certainly.